### PR TITLE
FOGL-2267 displayName added for ocs and pi_server plugins

### PR DESCRIPTION
**Pull request by ashish-jabble**

### CODE REVIEW

File: C/plugins/north/PI_Server_V2/plugin.cpp [hunk 1]
```diff
@@ -37,45 +37,45 @@ using namespace rapidjson;
...
 			"\"OMFMaxRetry\": { " \
 				"\"description\": \"Max number of retries for the communication with the OMF PI Connector Relay\", " \
 				"\"type\": \"integer\", \"default\": \"3\", " \
-				"\"order\": \"10\" }, " \
+				"\"order\": \"10\", \"displayName\": \"Maximum Retry\" }, " \
 			"\"OMFHttpTimeout\": { " \
 				"\"description\": \"Timeout in seconds for the HTTP operations with the OMF PI Connector Relay\", " \
 				"\"type\": \"integer\", \"default\": \"10\", " \
-				"\"order\": \"13\" }, " \
+				"\"order\": \"13\", \"displayName\": \"Http Timeout\" }, " \
```
Comment (MarkRiddoch) [inline C/plugins/north/PI_Server_V2/plugin.cpp]: I would capitalise HTTP as this is how it is an acronym

File: C/plugins/north/ocs_V2/plugin.cpp [hunk 2]
```diff
@@ -38,45 +38,45 @@ using namespace rapidjson;
...
 			"\"OMFMaxRetry\": { " \
 				"\"description\": \"Max number of retries for the communication with OCS\", " \
 				"\"type\": \"integer\", \"default\": \"3\", " \
-				"\"order\": \"10\" }, " \
+				"\"order\": \"10\", \"displayName\" : \"Maximum Retry\" }, " \
 			"\"OMFHttpTimeout\": { " \
 				"\"description\": \"Timeout in seconds for the HTTP operations with OCS\", " \
 				"\"type\": \"integer\", \"default\": \"10\", " \
-				"\"order\": \"13\" }, " \
+				"\"order\": \"13\", \"displayName\" : \"Http Timeout\" }, " \
```
Comment (MarkRiddoch) [inline C/plugins/north/ocs_V2/plugin.cpp]: As above HTTP rather than Http

File: python/foglamp/plugins/north/ocs/ocs.py [hunk 3]
```diff
@@ -134,76 +138,88 @@
...
+        "order": "10",
+        "displayName": "Maximum Retry"
     },
     "OMFHttpTimeout": {
         "description": "Timeout in seconds for the HTTP operations with the OMF PI Connector Relay",
         "type": "integer",
         "default": "30",
-        "order": "13"
+        "order": "13",
+        "displayName": "Http Timeout"
```
Comment (MarkRiddoch) [inline python/foglamp/plugins/north/ocs/ocs.py]: HTTP rather than Http

File: python/foglamp/plugins/north/pi_server/pi_server.py [hunk 4]
```diff
@@ -122,50 +126,58 @@
...
+        "order": "10",
+        "displayName": "Maximum Retry"
     },
     "OMFHttpTimeout": {
         "description": "Timeout in seconds for HTTP operations with the OMF PI Connector Relay",
         "type": "integer",
         "default": "10",
-        "order": "13"
+        "order": "13",
+        "displayName": "Http Timeout"
```
Comment (MarkRiddoch) [inline python/foglamp/plugins/north/pi_server/pi_server.py]: HTTP

File: python/foglamp/tasks/north/sending_process.py [hunk 5]
```diff
@@ -223,25 +223,29 @@ class SendingProcess(FoglampProcess):
...
+            "order": "7",
+            "displayName": "Duration Of Sending Process"
         },
         "blockSize": {
             "description": "Number of readings to send in each transmission",
             "type": "integer",
             "default": "5000",
-            "order": "8"
+            "order": "8",
+            "displayName": "Block Size Of Reading"
```
Comment (amarendra-dianomic) [inline python/foglamp/tasks/north/sending_process.py]: Transmission Block Size

File: python/foglamp/tasks/north/sending_process.py [hunk 6]
```diff
@@ -223,25 +223,29 @@ class SendingProcess(FoglampProcess):
             "description": "Time in seconds the sending process should run",
             "type": "integer",
             "default": "60",
-            "order": "7"
+            "order": "7",
+            "displayName": "Duration Of Sending Process"
```
Comment (amarendra-dianomic) [inline python/foglamp/tasks/north/sending_process.py]: Sending Process Spell

File: C/tasks/north/sending_process/sending.cpp [hunk 7]
```diff
@@ -70,14 +70,14 @@ static const string sendingDefaultConfig =
 		"the sending process.\", \"type\": \"boolean\", \"default\": \"true\" , \"readonly\": \"true\"  },"
 	"\"duration\": {"
 		"\"description\": \"How long the sending process should run (in seconds) before stopping.\", "
-		"\"type\": \"integer\", \"default\": \"60\" , \"order\": \"7\" }, "
+		"\"type\": \"integer\", \"default\": \"60\" , \"order\": \"7\", \"displayName\" : \"Duration Of Sending Process\" }, "
 	"\"blockSize\": {"
 		"\"description\": \"The size of a block of readings to send in each transmission.\", "
-		"\"type\": \"integer\", \"default\": \"500\", \"order\": \"8\"   }, "
+		"\"type\": \"integer\", \"default\": \"500\", \"order\": \"8\", \"displayName\" : \"Block Size Of Reading\" }, "
```
Comment (amarendra-dianomic) [inline C/tasks/north/sending_process/sending.cpp]: Transmission Block Size

File: C/tasks/north/sending_process/sending.cpp [hunk 8]
```diff
@@ -70,14 +70,14 @@ static const string sendingDefaultConfig =
 		"the sending process.\", \"type\": \"boolean\", \"default\": \"true\" , \"readonly\": \"true\"  },"
 	"\"duration\": {"
 		"\"description\": \"How long the sending process should run (in seconds) before stopping.\", "
-		"\"type\": \"integer\", \"default\": \"60\" , \"order\": \"7\" }, "
+		"\"type\": \"integer\", \"default\": \"60\" , \"order\": \"7\", \"displayName\" : \"Duration Of Sending Process\" }, "
```
Comment (amarendra-dianomic) [inline C/tasks/north/sending_process/sending.cpp]: Sending Process Spell

**Pull request merged**